lib/v2/screens/dashboard/transcations/interactor/viewmodels/transactions_list_bloc.dart

```diff
@@ -1,9 +1,9 @@
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:seeds/v2/domain-shared/page_state.dart';
-import 'package:seeds/v2/screens/dashboard/interactor/mappers/transactions_state_mapper.dart';
-import 'package:seeds/v2/screens/dashboard/interactor/usecases/load_transactions_use_case.dart';
-import 'package:seeds/v2/screens/dashboard/interactor/viewmodels/transactions_list_events.dart';
-import 'package:seeds/v2/screens/dashboard/interactor/viewmodels/transactions_list_state.dart';
+import 'package:seeds/v2/screens/dashboard/transcations/interactor/viewmodels/transactions_list_events.dart';
```

Comment:
transcations is not a word
`transactions`

Reply:
fix